src/System.Security.Cryptography.Algorithms/tests/ECDiffieHellmanTests.cs

```diff
@@ -35,6 +35,24 @@ public static void ECCurve_ctor()
             }
         }
 
+        [Theory]
+        [InlineData("1.3.132.0.35", "ECDSA_P521", 521)]
+        [InlineData("1.3.132.0.34", "ECDSA_P384", 384)]
+        [InlineData("1.2.840.10045.3.1.7", "ECDSA_P256", 256)]
+        public static void ECCurve_ctor_ECDSA_Algorithm_Name(string oidValue, string oidFriendlyName, int expectedKeySize)
```

Comment:
Unclear to me if this is the right place for this test.

Reply:
If the test just changes to `ECDiffieHellman.Create(ECCurve.CreateFromValue("1.3.132.0.35")))` (and the other two OIDs) then this is fine, but I'd possibly rename it to `Create_From_ECCurve_CreateFromValue`, or something of the sort.

Perhaps a second one for `Create_From_ECCurve_CreateFromFriendlyName_ECDSA_Identifier` (or something like that)